multidict/_multidict.pyx

```diff
@@ -238,7 +239,7 @@ cdef class MultiDictProxy(_Base):
         """Return a copy of itself."""
         return self._base_class(self)
 
-abc.Mapping.register(MultiDictProxy)
+MultiMapping.register(MultiDictProxy)
```

Comment:
IMHO using it as a class decorator would look cleaner

Reply:
Is that possible in Cython?

Reply:
cython must support decorators

Reply:
What decorators are your talking about?
Cython doesn't support deriving from abc classes IIRC.
`ABC.register()` is a standard way for registering C Extension classes

Reply:
What about
```python
@ABC.register
class ...:
```
?

Reply:
Aaah. Ok.

Reply:
Cython cannot apply decorators to cdef classes :(

Reply:
That's what I expected.